lib/models/builder.js

```diff
@@ -209,6 +210,12 @@ class Builder extends CoreObject {
       }
     }
 
+    this.ui.startProgress(progress.format(progress()));
+
+    const interval = setInterval(() => {
```

Comment:
Can you continue the convention of using `let` please?

Reply:
For my own knowledge, why do we prefer let over const?

Reply:
I believe `const` is appropriate here.

Reply:
Specifically, `interval` being changed later on, would in-fact break this algorithm.